smoke-testNuGet.Config

```diff
@@ -4,9 +4,6 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="source-built-packages" value="SOURCE_BUILT_PACKAGES" />
-    <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="dotnet/feed/dotnet-core" value="https://dotnetmygetlegacy.blob.core.windows.net/dotnet-core/index.json" />
-    <add key="ProdCon" value="PRODUCT_CONTRUCTION_PACKAGES" />
-    <add key="smoke-test feed" value="SMOKE_TEST_PACKAGE_FEED" />
```

Comment:
Are these "template" feeds needed for some smoke-test scenarios? Maybe track along with the ProdCon feed?

Reply:
It looks like this change is the same across the branches. Curious what will happen in CI.

Reply:
It worked internally, I think dotnet-public includes all the feeds we need for right now.  I will add a note that this may need to be changed to that same issue.